### ISSUE TITLE
Fix wpull.util_test.TestUtil

### ISSUE THREAD

**Issue by ivan** [ArchiveTeam/ludios_wpull#12]
```
======================================================================
FAIL: test_get_exception_message (wpull.util_test.TestUtil)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/grab/wpull/thematrix/wpull/util_test.py", line 95, in test_get_exception_message
    'NoNameservers', get_exception_message(NoNameservers())
AssertionError: 'NoNameservers' != 'All nameservers failed to answer the query.'
- NoNameservers
+ All nameservers failed to answer the query.
```

**Comment by HeliosLHC** [comment]
Fixed in https://github.com/ArchiveTeam/ludios_wpull/pull/25

Test failure was due to a change in the dnspython library for exception handling

**Comment by HeliosLHC** [comment]
Fixed in 5.0.0a
